src/main/java/io/confluent/examples/streams/TopArticlesLambdaExample.java

```diff
@@ -229,7 +232,10 @@ static KafkaStreams buildTopArticlesStream(final String bootstrapServers,
           return queue;
         },
 
-        new PriorityQueueSerde<>(comparator, valueAvroSerde)
+        // TODO: need to change this, after Damian's PR got merged to AK
```

Comment:
What needs to be changed here?  I'd also reference the actual PR, not just "Damian's PR". :)

Reply:
https://github.com/apache/kafka/pull/4009

Reply:
I meant: Do not leave the comment like this.  If we keep any such comment at all, then the comment should say sth like:

```java
// TODO: Code below must be changed to <short summary of the change> once KAFKA-5651 is merged.
```

Reply:
The comment is just for now (and for the reviewers) -- I will not merge until I can resolve this TODO :)

Reply:
Aaaah, I see.  Then all is good.  I didn't see a "[DON'T MERGE YET]" in the PR title.

Reply:
For docs PR, everyone merge by himself, thus not need to put into the title as I know it's not ready to get merged :) -- It's not AK where we need a committer to merge.